cfme/services/catalogs/catalog_item.py

```diff
@@ -1,107 +1,192 @@
-from utils import version, fakeobject_or_object
-from utils.appliance import Navigatable
-from utils.appliance.implementations.ui import CFMENavigateStep, navigate_to, navigator
+from cfme.web_ui import tabstrip
+from navmazing import NavigateToAttribute, NavigateToSibling
 from utils.update import Updateable
 from utils.pretty import Pretty
-from utils.version import current_version
-
-cfg_btn = partial(tb.select, "Configuration")
-policy_btn = partial(tb.select, "Policy")
-
-accordion_tree = partial(accordion.tree, "Catalog Items")
-dynamic_tree = Tree("//div[@id='basic_info_div']//ul[@class='dynatree-container']")
-entry_tree = BootstrapTreeview('automate_treebox')
-listview_table = CheckboxTable(table_locator='//div[@id="list_grid"]/table')
-
-template_select_form = Form(
-    fields=[
-        ('template_table', Table('//div[@id="prov_vm_div"]/table')),
-        ('add_button', form_buttons.add),
-        ('cancel_button', form_buttons.cancel)
-    ]
-)
-
-# Forms
-basic_info_form = Form(
-    fields=[
-        ('name_text', Input("name")),
-        ('description_text', Input("description")),
-        ('display_checkbox', Input("display")),
-        ('select_catalog', AngularSelect('catalog_id')),
-        ('select_dialog', AngularSelect('dialog_id')),
-        ('select_orch_template', AngularSelect('template_id')),
-        ('select_provider', AngularSelect('manager_id')),
-        ('select_config_template', AngularSelect('template_id')),
-        ('field_entry_point', Input("fqname")),
-        ('retirement_entry_point', Input("retire_fqname")),
-        ('edit_button', form_buttons.save),
-        ('apply_btn', '//a[normalize-space(.)="Apply"]')
-    ])
-
-# TODO: Replace with Taggable
-edit_tags_form = Form(
-    fields=[
-        ("select_tag", AngularSelect('tag_cat')),
-        ("select_value", AngularSelect('tag_add'))
-    ])
-
-detail_form = Form(
-    fields=[
-        ('long_desc', Input('long_description')),
-    ])
-
-resources_form = Form(
-    fields=[
-        ('choose_resource', Select("//select[@id='resource_id']")),
-        ('add_button', form_buttons.add),
-        ('save_button', form_buttons.save)
-    ])
-
-button_group_form = Form(
-    fields=[
-        ('btn_group_text', Input("name")),
-        ('btn_group_hvr_text', Input("description")),
-        ('add_button', form_buttons.add)
-    ])
-
-button_form = Form(
-    fields=[
-        ('btn_text', Input("name")),
-        ('btn_hvr_text', Input("description")),
-        ('select_dialog', Select("//select[@id='dialog_id']")),
-        ('system_process', Select("//select[@id='instance_name']")),
-        ('request', Input("object_request")),
-        ('add_button', form_buttons.add)
-    ])
-
-match_page = partial(match_location, title='Catalogs', controller='catalog')
+from utils.appliance import Navigatable
+from utils.appliance.implementations.ui import navigator, CFMENavigateStep, navigate_to
+
+from utils import version
```

Comment:
**Required**
bump this up a line (remove empty line) and move the navmazing import up with the other 3rd party imports.  Remove empty newlines below as well so all these imports are consistently formatted.